vtest/vtest_context.json

```diff
@@ -1,6 +1,6 @@
 {
     "mscore_ref_bin": "./build.debug/install/bin/mscore",
-    "score_dir": "./vtest/scores",
+    "score_dir": "/home/igor/Dev/scores",
```

Comment:
At the moment, the system is still in a state of development, it does not run on CI, so it is not critical, I will fix it in the next PR